core/src/main/scala/com/raphtory/algorithms/api/Table.scala

```diff
@@ -40,4 +40,5 @@ abstract class Table {
   def filter(f: Row => Boolean): Table
   def explode(f: Row => List[Row]): Table
   def writeTo(outputFormat: OutputFormat, jobName: String = ""): QueryProgressTracker
```

Comment:
We can remove the default value for jobName so that the two parameters version is also accessible from Python

Reply:
will do! then i will have to change all examples so they don't break.